packages/api-headless-cms-import-export/src/graphql/index.ts

```diff
@@ -7,15 +7,20 @@ import type { NonEmptyArray } from "@webiny/api/types";
 import { CmsModel } from "@webiny/api-headless-cms/types";
 
 export const attachHeadlessCmsImportExportGraphQL = async (context: Context): Promise<void> => {
+    const tenant = context.tenancy.getCurrentTenant();
+    const locale = context.i18n.getContentLocale();
     const models = await listModels(context);
 
-    if (models.length === 0) {
+    if (!tenant || !locale || models.length === 0) {
         return;
     }
 
     const plugin = new CmsGraphQLSchemaPlugin<Context>({
         typeDefs: createTypeDefs(models as NonEmptyArray<CmsModel>),
-        resolvers: createResolvers(models as NonEmptyArray<CmsModel>)
+        resolvers: createResolvers(models as NonEmptyArray<CmsModel>),
+        isApplicable: context =>
```

Comment:
I dont understand this check. You get tenant on line 10 and then you get it here again, with same method. Isn't it the same?